editor/src/clj/editor/app_view.clj

```diff
@@ -792,32 +935,22 @@
         (workspace/save-build-cache! workspace)
         build-results))))
 
-(defn- cached-build-target-output? [node-id label evaluation-context]
-  (and (= :build-targets label)
-       (project/project-resource-node? node-id evaluation-context)))
-
-(defn- update-system-cache-build-targets! [evaluation-context]
-  ;; To avoid cache churn, we only transfer the most important entries to the system cache.
-  (let [pruned-evaluation-context (g/pruned-evaluation-context evaluation-context cached-build-target-output?)]
-    (g/update-cache-from-evaluation-context! pruned-evaluation-context)))
-
 (defn- build-handler [project workspace prefs web-server build-errors-view main-stage tool-tab-pane]
   (let [project-directory (io/file (workspace/project-path workspace))
-        evaluation-context (g/make-evaluation-context)
```

Comment:
We can't grab an `evaluation-context` here and use it for the duration of the build, since it is a snapshot and any changes made by pre-/post-build hooks will not be seen in the `evaluation-context`.